plugin/storage/cassandra/factory.go

```diff
@@ -133,8 +133,7 @@ func (f *Factory) configureFromOptions(o *Options) {
 
 // Initialize implements storage.Factory
 func (f *Factory) Initialize(metricsFactory metrics.Factory, logger *zap.Logger) error {
-	f.primaryMetricsFactory = metricsFactory.Namespace(metrics.NSOptions{Name: "cassandra", Tags: nil})
-	f.archiveMetricsFactory = metricsFactory.Namespace(metrics.NSOptions{Name: "cassandra-archive", Tags: nil})
```

Comment:
Why not keep these two here? Otherwise you're duplicating namespace assignments twice, which means they can get out of sync.

Reply:
@yurishkuro Done. However, `CreateSamplingStore` (https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/cassandra/factory.go#L211) and `https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/cassandra/factory.go#L175` will now have the `role=primary` attached to the metrics they emit. Is this fine? or should they just be emitting metrics under the namespace passed into the storage factory without the role tag?

Reply:
I think sampling store should be `kind=cassandra, role=sampling`

Reply:
@yurishkuro Done. What about the dependency reader? I'm using the base factory for now.

Reply:
what would it look like?  Dependencies technically were always bundled within the spanstore, not a different "role".

Reply:
the metrics would be published under `jaeger_storage_***` with `kind=cassandra` but no `role` tag.

Reply:
that's fine

Reply:
although since it's using the primary connection / session I would pass it the primary metrics

Reply:
done